container/container.go

```diff
@@ -484,7 +484,9 @@ func copyEscapable(dst io.Writer, src io.ReadCloser, keys []byte) (written int64
 		nr, er := src.Read(buf)
 		if nr > 0 {
 			// ---- Docker addition
+			preservBuf := []byte{}
```

Comment:
Trying to follow the flow here, but it seems to me that "preservBuf" shouldn't be needed since it MUST match the keys, right?  meaning if we're on the 3rd char of 'keys' then preservBuf must be the same as the first 2 chars of keys, right?